app/scripts/modules/appengine/instance/details/details.controller.ts

```diff
@@ -76,7 +80,11 @@ class AppengineInstanceDetailsController {
   }
 
   private autoClose(): void {
-    if (!(this.$scope as any).$$destroyed) {
```

Comment:
I would remove this method altogether, and change the `this.autoClose()` call above to just set `this.state.loading = false`